prog/vnet/load_balancer_health_probes.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class Prog::Vnet::LoadBalancerHealthProbes < Prog::Base
```

Comment:
I think you can defer most (all?) of this logic to monitoring by defining `check_pulse` and `init_health_monitor_session` for LoadBalancerVms.

Reply:
Possible, but, initially, I wanted to keep this logic isolated as much as possible. It's a good addition for revision 2.